purchase_quotation_products/views/product_product_views.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="product_product_tree_view" model="ir.ui.view">
+        <field name="name">product.product.form</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_product_tree_view"/>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <tree position="attributes">
+                <attribute name="edit">true</attribute>
+                <attribute name="create">false</attribute>
+                <attribute name="editable">top</attribute>
+            </tree>
+            <xpath expr="//tree">
+                <button name="action_product_form" type="object" icon="fa-external-link" string="Open Product Form View"/>
+            </xpath>
+             <xpath expr="//tree//field[1]" position="before">
+                <field name="qty_purchase"/>
+            </xpath>
+            <field name="lst_price" position="replace">
+                <field name="standard_price" readonly="1"/>
+            </field>
+            <field name="default_code" position="attributes">
+                <attribute name="readonly">1</attribute>
+            </field>
+            <xpath expr="//field[contains(@name,'name')]" position="attributes">
```

Comment:
> +            <xpath expr="//field[contains(@name,'name')]" position="attributes">

I imagine that something goes wrong applying some rebase since this change was already fixed and now is back as it was in first place. Please  dont use `field[contains(@name,'name)]` instead use `field[@name='name"]`

Apply this to all this file